src/field_theory/krull_topology.lean

```diff
@@ -198,6 +202,7 @@ section krull_t2
 
 open_locale topological_space filter
 
+-- TODO: move this to a more general file
```

Comment:
This must have slipped through in an earlier PR, but this lemma is in the wrong spot.

Reply:
Do I need to do anything about it or is it something to be resolved after the PR gets merged?

Reply:
If you are happy with the other changes that I pushed, please move this lemma to an appropriate place. After that, I think this is ready for merging.